_data/overviews.yml

```diff
@@ -2,53 +2,6 @@
 - category: Standard Library
   description: "Guides and overviews covering the Scala standard library."
   overviews:
-    - title: Collections
```

Comment:
These diffs done in other PRs should not be included to decrease review labor.
Could you rebase onto `master` branch ?

Reply:
日本語ですみません。
ブランチ`translation-tour-ja`にて、`git rebase master`を実施したところ、今までの私のコミットが全て競合した勢いでコンフリクトが発生しました。
やり方が間違ってるのでしょうか？

Reply:
https://github.com/scala/docs.scala-lang/pull/1387/commits/62c3f0b12815c622478cf7213554bd45f78a3a44 での merge をキャンセルするために、(他に一切何も commit して無ければ) reset --hard HEAD^ してから、rebase してみるとうまくいくと思います。

```
git reset --hard HEAD^
git rebase master
```

Reply:
こんな感じになるはずです https://github.com/scala/docs.scala-lang/compare/master...eed3si9n:wip/tour-ja?expand=1

Reply:
@eed3si9n 
`(他に一切何も commit して無ければ) `ってわざわざ書いていただいていたのに、こちらでcommit作業をしてしまいました。申し訳ないです。

```
git reset --hard 386a7c01
git cherry-pick 62c3f0b1..bd366b47
git rebase master
```

一応、上記のコマンドでrebaseまでは済んだのですが、以下のエラーが出ます。
リポジトリ的に整合性取れなくなっているという認識なので、`git push -f`しようと思うのですが、問題ないでしょうか？
```
 ! [rejected]          translation-tour-ja -> translation-tour-ja (non-fast-forward)
error: failed to push some refs to 'https://github.com/take2webservice/docs.scala-lang.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Reply:
遠慮無く git push -f してokだと思います。

Reply:
@eed3si9n 
ありがとうございます！
psuhしました！

@exoego 
最新masterにリベースしたものをpushし、ビルド通りました！